packages/web3-core-method/src/index.js

```diff
@@ -883,7 +883,11 @@ function _handleTxPricing(method, tx) {
                         maxFeePerGas = tx.gasPrice;
                         delete tx.gasPrice;
                     } else {
-                        maxPriorityFeePerGas = tx.maxPriorityFeePerGas || '0x9502F900'; // 2.5 Gwei
+                        maxPriorityFeePerGas = tx.maxPriorityFeePerGas 
+                         || (
+                             method.requestManager.provider.defaultOnlineGasPrice ? gasPrice
```

Comment:
Please share some reference to this change, why not we use `defaultOnlineGasPrice` instead of `gasPrice` here?

Reply:
Gasprice is a dynamic value and cannot be fixed. In the experiment, automatic needs 30-35.

Reply:
@0123cf Please share some reference for this change. Would be nice if you can add that reference in the code comment with explanation for future use.

Reply:
@0123cf

Reply:
@0123cf Sorry to be iterating on this feedback. Your provided reference is pointing a comment on our Repo. I was looking to have more official documentation reference from Polygon. Which explains about `defaultOnlineGasPrice` property and where it should be used.